pkg/icingaredis/heartbeat.go

```diff
@@ -25,6 +25,7 @@ type Heartbeat struct {
 	active         bool
 	events         chan *HeartbeatMessage
 	lastReceivedMs int64
+	lastMessageMs  int64
```

Comment:
Am I missing something? Why can't you make this of type `atomic.Int64` and use its `Load()` and `Store()` methods?

Reply:
It was implemented this way to be symmetrical to `lastReceivedMs`, especially in the later calls.

Reply:
The previous implementation was designed that way because `atomic.Int64` was not available prior to [Go 1.19](https://go.dev/doc/go1.19#atomic_types).

Reply:
Okay, but it works, right? There is no necessity to change it, which would blow up this change, btw. And having these two implemented differently would just look odd.

Reply:
> There is no necessity to change it, which would blow up this change, btw. And having these two implemented differently would just look odd.

I was referring to the new type you introduced and did not mean for you to change the existing code. Personally, we should be more focused on using new features rather than maintaining symmetry with the existing code :), anyway I've changed the existing usages with #829.

Reply:
Personally, I don't think always using the latest features is something to strive for, unless it adds real value. In that case, the API is more foolproof. I am for changing it, as you did, but then it should be done consequentially. I will update this PR as well.